### PR TITLE
BBC Scotland SVG

### PR DESCRIPTION
**Pull request by sareh**
No ticket

Screenshot BBC Scotland logo in storybook

Storybook link: http://localhost:8180/?path=/story/utilities-svgs--scotland

**Overall change:** Add Scotland SVG needed for banner on `/scotland/articles/*`

**Code changes:**

- Add optimised BBC Scotland SVG.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/utilities/psammead-assets/README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ import { AMP_SCRIPT } from '@bbc/psammead-assets/amp-boilerplate';
 
 Service brand SVGs, like `news`, are objects that contains an svg group, viewbox, ratio and height;
 
-This package currently has brand SVGs for the BBC services `igbo`, `news`, `persian`, `pidgin`, `thai` and `yoruba`.
+This package currently has brand SVGs for the BBC News World Services as well as for BBC News, BBS Scotland.
```
Comment (dr3) [inline packages/utilities/psammead-assets/README.md]:
```suggestion
This package currently has brand SVGs for the BBC News World Services as well as for BBC News, BBC Scotland.
```

Comment (sareh) [inline packages/utilities/psammead-assets/README.md]:
```suggestion
This package currently has brand SVGs for the BBC News World Services as well as for BBC News, BBC Scotland.
```

**Review by dr3 — APPROVED** [review]

**Review by RayNjeri — APPROVED** [review]

**Comment by sareh** [comment]
Confirmed with Jim - this PR is good to be merged. It'll be fully UX reviewed when integrated into Simorgh.

**Comment by greenc05** [comment]
Was this checked in high contrast mode and also changing the colour preferences in FF @sareh? I assume that we also have some visually hidden text for it?

**Pull request merged**